### PR TITLE
[PHEE-652] Mojaloop kafka not consuming data

### PR DESCRIPTION
**Pull request by abhinavm117**

### CODE REVIEW

File: helm/example-backend/Chart.yaml
Comment (fynmanoj) [inline helm/example-backend/Chart.yaml:1]: why this chart is needed? 
cant we do these changes in superset chart?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
requested clarification

File: helm/example-backend/Chart.yaml
Comment (abhinavm117) [reply helm/example-backend/Chart.yaml:1]: Superset chart is not allowing us to override these values. Also in mojaloop documentation it is recommended to use separate chart with different deployment.

**Review by abhinavm117 — COMMENTED** [review]
Update Readme order of instructions

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**